src/physics/model.h

```diff
@@ -38,14 +39,67 @@ class ModelBase
     	const std::array<dealii::Tensor<1,dim,real>,nstate> &solution_gradient,
         const dealii::types::global_dof_index cell_index) const = 0;
 
-    /// Source terms additional to the baseline physics
+    /// Convective Numerical Split Flux for split form additional to the baseline physics
+    virtual std::array<dealii::Tensor<1,dim,real>,nstate> 
+    convective_numerical_split_flux (
+        const std::array<real,nstate> &soln_const, 
+        const std::array<real,nstate> &soln_loop) const = 0;
+
+    /// Spectral radius of convective term Jacobian.
+    /** Used for scalar dissipation
+     */
```

Comment:
```suggestion
    /** Used for scalar dissipation */
```

Reply:
fixed

Reply:
apply this throughout the header file

Reply:
Do you mean for the one-line comment, */ should be at the end of the comment at the same line?

Reply:
fixed.
All comments with one line have the suggested structure, but the comments with multiple lines still maintain the old structure (with */ in the new line).